wallaby.js

```diff
@@ -1,91 +1,16 @@
 /* eslint-disable import/no-commonjs */
-const babelConfig = require("./build-settings/babel.config.js");
-
-module.exports = function (wallaby) {
```

Comment:
Does the change mean that all the contents of this function are now handled with `autoDetect=true`?

Reply:
I think it looks at jest.config.js and figures things out from that.